### ISSUE TITLE
AppImage crashing on Linux Mint

### ISSUE THREAD

**Issue by gagarin55** [cypherstack/stack_wallet#508]
Can't run latest AppImage build on Linux:
```
$ ./sw-v1.7.4.AppImage 
flutter: Log: [Info][2023-04-29 11:46:28.494Z]: Logs older than 2023-03-30 11:46:28.474634Z cleared!
flutter: BUILD: _MaterialAppWithThemeState
double free or corruption (!prev)
Aborted (core dumped)

```
I have

```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Linuxmint
Description:	Linux Mint 21
Release:	21
Codename:	vanessa
```

Any advice ?
Thank you

**Comment by gagarin55** [comment]
Same for 1.7.8
```
$ ./sw-v1.7.8.AppImage 
flutter: Log: [Info][2023-05-20 15:06:31.956Z]: Logs older than 2023-04-20 15:06:31.942274Z cleared!
flutter: Log: [Info][2023-05-20 15:06:34.223Z]: Installing default light theme...
flutter: Log: [Info][2023-05-20 15:06:34.260Z]: Installing default light theme... finished
flutter: Log: [Info][2023-05-20 15:06:34.265Z]: Installing default dark theme... 
flutter: Log: [Info][2023-05-20 15:06:34.308Z]: Installing default dark theme... finished
flutter: BUILD: _MaterialAppWithThemeState
double free or corruption (!prev)
Aborted (core dumped)
```

**Comment by gagarin55** [comment]
It fixed in v1.7.15
